src/renderers/settings.ts

```diff
@@ -64,6 +64,15 @@ export class RendererSettings {
 	enableSvg: boolean;
 
 	/**
+	 * Fallback fonts list to be used when font from ASS Styles not loaded.
+	 *
+	 * Defaults to 'Arial, Helvetica, sans-serif'.
+	 *
+	 * @type {string}
+	 */
+	fallbackFonts: string;
```

Comment:
It could either be a string, in which case the user has to be sure they pass in a string that's a valid CSS font-family property (i.e., comma-separated, and individual names in double quotes if necessary).

Or, it could be a string array, and libjass could do ```fallbackFonts.map(fontName => `"${ fontName }"`).join(", ")``` at the few places it would use it. Although this has the problem that things like sans-serif cannot be quoted as they are keywords (actually this is a bug with current libjass also).

The first option (what you currently have) is fine, but then you should clarify that in the doc comment.